conda_forge_tick_action/automerge.py

```diff
@@ -26,17 +26,40 @@
 GOOD_MERGE_STATES = ["clean", "has_hooks", "unknown", "unstable"]
 
 
+# https://stackoverflow.com/questions/6194499/pushd-through-os-system
+@contextlib.contextmanager
+def pushd(new_dir):
+    previous_dir = os.getcwd()
+    os.chdir(new_dir)
+    try:
+        yield
+    finally:
+        os.chdir(previous_dir)
+
+
+def _get_conda_forge_config(pr):
+    """get the conda-forge.yml from upstream master
+
+    We always do this to make sure we use the maintainer settings and not
+    any from a fork.
+    """
+    with tempfile.TemporaryDirectory() as tmpdir:
+        subprocess.run(
+            ["git", "clone", pr.base.repo.clone_url, tmpdir],
```

Comment:
Shouldn't this use the target branch as well?

Reply:
Ah the target branch of the pr. Yes it should.

Reply:
done.